bin/pyspark

```diff
@@ -72,6 +65,4 @@ if [[ -n "$SPARK_TESTING" ]]; then
   exit
 fi
 
-export PYSPARK_DRIVER_PYTHON
```

Comment:
Why did we move this two?

Reply:
I moved the`PYSPARK_PYTHON` and `PYSPARK_DRIVER_PYTHON` exports closer together to make the whole script more coherent